### ISSUE TITLE
`nil?` method with argument doesn't work

### ISSUE THREAD

**Issue by AnatolyShirykalov** [dry-rb/dry-logic#40]
Can't use version 0.5.0. Method `nil?` was called without an argument.

```
ArgumentError: wrong number of arguments (given 0, expected 1)
  /home/anatoly/.gem/ruby/2.6.0/gems/dry-logic-0.5.0/lib/dry/logic/predicates.rb:17:in `nil?'
  /home/anatoly/.gem/ruby/2.6.0/gems/dry-configurable-0.7.0/lib/dry/configurable.rb:110:in `setting'
  /home/anatoly/.gem/ruby/2.6.0/gems/dry-validation-0.12.2/lib/dry/validation/schema/class_interface.rb:15:in `<class:Schema>'
  /home/anatoly/.gem/ruby/2.6.0/gems/dry-validation-0.12.2/lib/dry/validation/schema/class_interface.rb:7:in `<module:Validation>'
  /home/anatoly/.gem/ruby/2.6.0/gems/dry-validation-0.12.2/lib/dry/validation/schema/class_interface.rb:6:in `<module:Dry>'
  /home/anatoly/.gem/ruby/2.6.0/gems/dry-validation-0.12.2/lib/dry/validation/schema/class_interface.rb:5:in `<top (required)>'
```

**Comment by solnic** [comment]
@AnatolyShirykalov this will be fixed in dry-validation 0.13.0. It'll be released later today. Sorry about the trouble.

**Comment by solnic** [comment]
Please update to dry-validation 0.13.0. It's fixed there.
